README.md

```diff
@@ -1,86 +1,126 @@
-The official Docker images for the .NET Framework on Windows Server Core.
+# Supported tags and respective `Dockerfile` links
 
-# .NET Framework Containers
+-       [`4.6.2`, `latest` (*4.6.2/Dockerfile*)](4.6.2/Dockerfile)
+-       [`3.5` (*3.5/Dockerfile*)](3.5/Dockerfile)
 
-There are multiple flavors of the .NET Framework Docker images available, for both Web and Console application workloads.
+For more information about these images and their history, please see [the relevent Dockerfile (`microsoft/dotnet-framework-docker`)](https://github.com/microsoft/dotnet-framework-docker/search?utf8=%E2%9C%93&q=FROM&type=Code). These images are updated via [pull requests to the `microsoft/dotnet-framework-docker` GitHub repo](https://github.com/dotnet/dotnet-docker/pulls?utf8=%E2%9C%93&q=).
```

Comment:
This link incorrectly points to dotnet-docker - https://github.com/dotnet/dotnet-docker/pulls?utf8=%E2%9C%93&q=